### PR TITLE
Dedupe log

### PR DESCRIPTION
**Pull request by Eijebong**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/22388)

### CODE REVIEW

**Comment by Eijebong** [comment]
I updated libc, should be fixed

**Comment by jdm** [comment]
Mmm, we may have broken gaol compilation on android by merging https://github.com/servo/gaol/pull/51.

**Comment by jdm** [comment]
@Eijebong Note that the magic leap and both arm builds were also broken for similar reasons in the last push.

**Comment by Eijebong** [comment]
Urgh logs expired and I don't remember what was wrong...

@bors-servo try=android

**Comment by Eijebong** [comment]
@bors-servo try=android

Stop ignoring me !

**Comment by Eijebong** [comment]
(╯°□°）╯ sɹoq

**Comment by Eijebong** [comment]
@bors-servo try=android

That was dumb...

**Comment by Eijebong** [comment]
@jdm Ok, disabled gaol on android now

**Comment by jdm** [comment]
@Eijebong It will also need to be disabled on arm32, arm64, and magic leap.

**Comment by Eijebong** [comment]
Ouch... Do we have try targets for that ?

**Comment by jdm** [comment]
try=magicleap and try=arm

**Comment by Eijebong** [comment]
Perfect then, let's see what needs to be changed then, (abusing try since I only have so many CPUs and am working on firefox and rustc stuff)

@bors-servo try=magicleap

**Comment by Eijebong** [comment]
Nice :)

@try=arm

**Comment by Eijebong** [comment]
@bors-servo try=arm

Should be green now

File: components/constellation/Cargo.toml [hunk 1]
```diff
@@ -45,5 +45,5 @@ servo_url = {path = "../url"}
 webvr_traits = {path = "../webvr_traits"}
 webrender_api = {git = "https://github.com/servo/webrender", features = ["ipc"]}
 
-[target.'cfg(all(not(target_os = "windows"), not(target_os = "ios")))'.dependencies]
+[target.'cfg(all(not(target_os = "windows"), not(target_os = "ios"), not(target_os="android"), not(target_arch="arm"), not(target_arch="aarch64")))'.dependencies]
```
Comment (jdm) [inline components/constellation/Cargo.toml]: I wonder if we should just do `any(target_os = "macos", target_os = "linux")` at this point :/

File: components/constellation/pipeline.rs [hunk 2]
```diff
@@ -571,7 +571,39 @@ impl UnprivilegedPipelineContent {
         }
     }
 
-    #[cfg(all(not(target_os = "windows"), not(target_os = "ios")))]
```
Comment (jdm) [inline components/constellation/pipeline.rs]: I think this breaks windows compilation?

**Review by jdm — CHANGES_REQUESTED** [review]

File: components/constellation/Cargo.toml (hunk 1 above)
Comment (Eijebong) [reply components/constellation/Cargo.toml]: Well Linux on arm is still Linux, right ?

Comment (jdm) [reply components/constellation/Cargo.toml]: Good point!

File: components/constellation/pipeline.rs (hunk 2 above)
Comment (jdm) [reply components/constellation/pipeline.rs]: Oh, I see. So now we have three spawn_multiprocess implementations? Delightful!

**Review by jdm — APPROVED** [review]

File: components/servo/lib.rs [hunk 3]
```diff
@@ -661,14 +673,26 @@ pub fn run_content_process(token: String) {
...
+#[cfg(any(
+    target_os = "windows",
+    target_os = "ios",
+    target_os = "android",
+    target_arch = "arm",
+    target_arch = "aarch64"
+))]
 fn create_sandbox() {
-    panic!("Sandboxing is not supported on Windows or iOS.");
+    panic!("Sandboxing is not supported on Windows, iOS, ARM targets and android.");
```
Comment (atouchet) [inline components/servo/lib.rs]: "Android" should probably be capitalized here.

File: components/constellation/pipeline.rs [hunk 4]
```diff
@@ -571,7 +571,39 @@ impl UnprivilegedPipelineContent {
...
+    #[cfg(any(
+        target_os = "android",
+        target_arch = "arm",
+        target = "aarch64"
```
Comment (Eijebong) [inline components/constellation/pipeline.rs]: oups, it's target_arch here...

**Pull request merged**